src/services/tech-record-document-conversion.ts

```diff
@@ -515,9 +484,32 @@ const upsertAxles = async (connection: Connection, techRecordId: any, techRecord
         );
 
         debugLog(`upsertTechRecords: Upserted axle (tech-record-id: ${techRecordId}, ID: ${axleUpsertResponse.rows.insertId})`);
+    }
+
+    return;
+};
+
+const upsertAuthIntoService = async (connection: Connection, techRecordId: string, techRecord: TechRecord): Promise<void> => {
+    debugLog(`upsertTechRecords: Upserting authIntoService (tech-record-id: ${techRecordId})...`);
 
-        insertedIds.push(axleUpsertResponse.rows.insertId);
+    if (!techRecord.authIntoService) {
+        debugLog(`upsertTechRecords: no authIntoService present`);
+        await deleteBasedOnWhereIn(AUTH_INTO_SERVICE_TABLE.tableName, "technical_record_id", [techRecordId], connection);
+        return;
     }
 
-    return insertedIds;
+    const response = await executeFullUpsert(
+        AUTH_INTO_SERVICE_TABLE,
+        [
+            techRecordId,
+            techRecord.authIntoService?.cocIssueDate,
+            techRecord.authIntoService?.dateAuthorised,
+            techRecord.authIntoService?.datePending,
+            techRecord.authIntoService?.dateReceived,
+            techRecord.authIntoService?.dateRejected,
+        ],
+        connection
+    );
```

Comment:
In regards to : https://github.com/dvsa/cvs-nop/pull/17/files#r1140293439

I think it looks good, and if VTM decides to have 1-to-M relationship this will in part address it - with minor change in the database schema.